ports-of-call/portable_arrays.hpp

```diff
@@ -187,12 +187,28 @@ class PortableMDArray {
   // l-value, e.g.: a(3) = 3.0;
   template <typename... Is> //, class = std::enable_if_t<(sizeof...(Is) > 3)>>
   PORTABLE_FORCEINLINE_FUNCTION constexpr reference operator()(Is... idxs) noexcept {
-    return pdata_[util::fast_findex({static_cast<size_type>(idxs)...}, nxs_, strides_)];
+    index_type idx;
+    if constexpr (sizeof...(Is) == 0) {
+      idx = 0;
+    } else if constexpr (sizeof...(Is) == 1) {
+      idx = get_first(idxs...);
+    } else {
+      idx = util::fast_findex({static_cast<size_type>(idxs)...}, nxs_, strides_);
```

Comment:
`fast_findex` already checks the size of the input indices. It's missing the empty case but it should be added there.

What is the difference between

```c++
fast_findex(A const &ijk, A const &dim, A const &stride) {
  constexpr auto N = get_size(A{});
  if constexpr (N == 1) {
    return ijk[0];
  }
 //...
}
``` 
and `get_first` ?

Reply:
You could instead modify fast index instead of adding get_first I think. Either should be fine.